### PR TITLE
Hide Plugins view

### PR DESCRIPTION
**Pull request by vitaliy-guliy**
Signed-off-by: Vitaliy Guliy <vgulyy@redhat.com>

### What does this PR do?
Hides the Plugins view by removing an appropriate View/Plugins menu item and unregistering a command toggling the view.

Unbinding of `PluginFrontendViewContribution` does not work with failing Theia.

Fixes issue https://github.com/eclipse/che/issues/13670

### CODE REVIEW

**Comment by slemeur** [comment]
related to https://github.com/eclipse/che/issues/13670 ?

**Comment by vitaliy-guliy** [comment]
@slemeur yes. I thought we have no issue for that. Thanks.

**Comment by vitaliy-guliy** [comment]
I will add commit renaming Che Plugins to plugins

File: extensions/eclipse-che-theia-plugin-ext/src/browser/plugin/che-plugin-menu.ts [hunk 1]
```diff
@@ -33,12 +34,26 @@ export class ChePluginMenu {
...
+     * TEMPORARY SOLUTION
+     *
+     * Following code removes 'View/Plugins' menu item and the command that displays/hides Plugins view.
+     * In the future we will try to refactor Che Plugins view and move it to the 'plugin-ext'.
+     */
+    constructor(
+        @inject(MenuModelRegistry) menuModelRegistry: MenuModelRegistry,
+        @inject(CommandRegistry) commandRegistry: CommandRegistry
+    ) {
+        menuModelRegistry.unregisterMenuAction('pluginsView:toggle', CommonMenus.VIEW_VIEWS);
```
Comment (benoitf) [inline extensions/eclipse-che-theia-plugin-ext/src/browser/plugin/che-plugin-menu.ts]: couldn't we just unbind the stuff from inversify module ?

Comment (vitaliy-guliy) [reply extensions/eclipse-che-theia-plugin-ext/src/browser/plugin/che-plugin-menu.ts]: Unbinding of PluginFrontendViewContribution does not work with failing Theia. I noticed that in a title.
And yes, it's an easiest and most proper way to unregister.

Comment (vitaliy-guliy) [reply extensions/eclipse-che-theia-plugin-ext/src/browser/plugin/che-plugin-menu.ts]: I have played with it a lot yesterday.
The problem that `PluginFrontendViewContribution` is binded as `Command`, `Keybinding` and as `Menu` contribution.
I don't know how we can unbind the contribution from our extension.

**Comment by evidolob** [comment]
@slemeur Should we include this in 7.0.0?

**Comment by sunix** [comment]
> @slemeur Should we include this in 7.0.0?

let's ask triage team to prioritize https://github.com/eclipse/che/issues/13670

**Pull request merged**